test/evp_kdf_test.c

```diff
@@ -1384,6 +1384,157 @@ static int test_kdf_kbkdf_fixedinfo(void)
 }
 #endif /* OPENSSL_NO_CMAC */
 
+static int test_kdf_kbkdf_kmac(void)
```

Comment:
Do we need this given we've got an evp_test?

Reply:
We do this for everything else - so why not. I find it easier to refer to this file than evp_test.. Example or demo could be done but I ran out of time :)
Pauli STOP WORKING and go on holidays :)

Reply:
Something to reconsider in the future.
evp_test is a better place and an example/demo would be good.